### PR TITLE
Document function `translateUTF8`

### LINKED ISSUE #2026: Document `translateUTF8` function.
Create documentation for the `translateUTF8` function. See #1833 for details on how to complete this issue.

Child issue of #1833.

### PR DESCRIPTION
**Pull request by peter279k**
Document function `translateUTF8`.

Fixes https://github.com/ClickHouse/clickhouse-docs/issues/2026

### Changelog category (leave one):

- Documentation (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
*This is an automated comment for commit b96543e57e39f021cd56d3936d93b9382a5c824d with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/62473/b96543e57e39f021cd56d3936d93b9382a5c824d/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: docs/en/sql-reference/functions/string-replace-functions.md [hunk 1]
```diff
@@ -193,3 +193,33 @@ Result:
...
+**Returned value**
+
+- `s`: A string type [String](/docs/en/sql-reference/data-types/string.md).
+
+**Examples**
+
+Query:
+
+``` sql
+SELECT translateUTF8('Hello, World!', 'delor', 'DELOR') AS res;
```
Comment (rschu1ze) [inline docs/en/sql-reference/functions/string-replace-functions.md:218]: An example which actually involves non-ASCII UTF-8 strings would be nice.

Maybe

```sql
SELECT translateUTF8('Münchener Straße', 'üß', 'us');

Result: Munchener Strase
```

Comment (peter279k) [reply docs/en/sql-reference/functions/string-replace-functions.md]: Updated. Thanks for your recommendation.

Comment (rschu1ze) [reply docs/en/sql-reference/functions/string-replace-functions.md]: (as a German, I must say that `Munchener Strase` is terrible spelling 😄)

**Pull request merged**